### PR TITLE
SOF-718 feat: Action Triggers

### PR DESCRIPTION
**Pull request by thomasslee97**
This PR migrates the shortcuts over to the new Action Triggers system.

The migrations added are:
- System: Remove all of the triggers auto-populated by core migrations (these are the shortcuts NRK use).
- AFVD/Offtube showstyle: These add te triggers used by TV 2, using the default config values for studio sources / DVE layouts to build the triggers for cutting / queuing / routing to aux / cutting to a DVE box.
  - These migrations run twice per showstyle, once for blueprints version `v1.7.0` (bumped by this PR) and once after the default migration steps, to auto-fill new triggers in the future when the blueprint defaults update.
  - These migrations will generate some "placeholder" triggers in certain cases. For example, if "route to box 3" shortcuts only exist for 5 cameras, but the defaults require 10 cameras, 5 "placeholder" triggers will be created which are not bound to any keyboard shortcuts, so that it is easy to bind these sources to hotkeys in the future.

I have followed the labels used by the keyboard map in AFVD for the names of actions so that waht's present in the settings UI matches what's seen in the keyboard map.

What follows is a summary of the keyboard shortcuts I have found during this work, which I will transfer to confluence when merging this PR.

## "Rundown View" shortcuts
These are shortcuts that control the rundown (e.g. Activate)

| Key                   | Action                |
| --------------------- | --------------------- |
| AnyEnter              | Take                  |
| Backtick	            | Activate Rundown      |
| Shift+ArrowUp	        | hist. op SKIP         |
| Shift+ArrowDown	    | hist. ned SKIP        |
| Shift+ArrowLeft	    | hist. venstre SKIP    |
| Shift+ArrowRight	    | hist. højre SKIP      |
| Shift+Home	        | gå til aktiv linje    |
| Shift+Escape	        | Reload rundown        |
| Ctrl+Backtick	        | øve rundown           |
| Ctrl+Home	            | REW tidslinje         |
| Ctrl+Shift+Backtick	| deaktivate rundown    |

## "Browser" Shortcuts
These are baked into the browser, so don't need to be created in Sofie

| Key            | Action          |
| ---------------| --------------- |
| Ctrl+R         |	Reload Browser |
| Shift+PageUp   |	UI max         |
| Shift+PageDown |	UI min         |

## "ShowStyle" Shortcuts
These are driven by the ShowStyle configuration

| Key            | Action          |
| ---------------| --------------- |
| F10	                        | Last DVE |
| Ctrl+Alt+Shift+KeyB	        | Last Live |
| KeyQ	                    | overlay ALT UD |
| KeyE	                    | EVS 1 VO |
| KeyR	                    | EVS 1 100% |
| KeyU	                    | EVS 2 VO |
| KeyI	                    | EVS 2 100% |
| KeyC	                    | 3 SPLIT |
| KeyV                    	| 2BM |
| KeyB	                    | 3BM |
| KeyN	                    | BM Ipad |
| KeyM	                    | SOM |
| Comma	                    | MB |
| Period	                    | BM |
| Shift+KeyE	                | EVS 1 100% inp 1 |
| Shift+KeyD	                | EVS 1 VO inp 1 |
| Shift+KeyA	                | Design Style SC |
| Minus	                    | STOP soundpl. |
| Shift+KeyU	                | EVS 2 VO inp 1 |
| Shift+KeyI	                | EVS 2 100% inp 1 |
| Shift+F1	                | KAM 1 inp 1 |
| Shift+F2	                | KAM 2 inp 1 |
| Shift+F3	                | KAM 3 inp 1 |
| Shift+F4	                | KAM 4 inp 1 |
| Shift+F5	                | KAM 5 inp 1 |
| Digit1	                    | LIVE 1 |
| Digit2	                    | LIVE 2 |
| Digit3	                    | LIVE 3 |
| Digit4	                    | LIVE 4 |
| Digit5	                    | LIVE 5 |
| Digit6	                    | LIVE 6 |
| Digit7	                    | LIVE 7 |
| Digit8	                    | LIVE 8 |
| Digit9	                    | LIVE 9 |
| Digit0	                    | LIVE 10 |
| Shift+Digit1	            | LIVE 1 inp 1 |
| Shift+Digit2	            | LIVE 2 inp 1 |
| Shift+Digit3	            | LIVE 3 inp 1 |
| Shift+Digit4	            | LIVE 4 inp 1 |
| Shift+Digit5	            | LIVE 5 inp 1 |
| Shift+Digit6	            | LIVE 6 inp 1 |
| Shift+Digit7	            | LIVE 7 inp 1 |
| Shift+Digit8	            | LIVE 8 inp 1 |
| Shift+Digit9	            | LIVE 9 inp 1 |
| Shift+Digit0	            | LIVE 10 inp 1 |
| Ctrl+Digit1	                | LIVE 1 inp 2 |
| Ctrl+Digit2	                | LIVE 2 inp 2 |
| Ctrl+Digit3	                | LIVE 3 inp 2 |
| Ctrl+Digit4	                | LIVE 4 inp 2 |
| Ctrl+Digit5	                | LIVE 5 inp 2 |
| Ctrl+Digit6	                | LIVE 6 inp 2 |
| Ctrl+Digit7	                | LIVE 7 inp 2 |
| Ctrl+Digit8	                | LIVE 8 inp 2 |
| Ctrl+Digit9	                | LIVE 9 inp 2 |
| Ctrl+Digit0	                | LIVE 10 inp 2 |
| Ctrl+F1	                    | KAM 1 inp 2 |
| Ctrl+F2	                    | KAM 2 inp 2 |
| Ctrl+F3	                    | KAM 3 inp 2 |
| Ctrl+Alt+Shift+KeyA	        | KAM 4 inp 2 |
| Ctrl+F5	                    | KAM 5 inp 2 |
| Ctrl+KeyE	                | EVS 1 100% inp 2 |
| Ctrl+KeyD	                | EVS 1 VO inp 2 |
| Ctrl+Alt+Shift+KeyI	        | EVS 2 VO inp 2 |
| Ctrl+KeyI	                | EVS 2 100% inp 2 |
| Ctrl+Alt+Shift+KeyE	        | mics OP |
| Ctrl+Alt+Shift+KeyD	        | mics NED |
| Shift+Ctrl+F1	            | KAM 1 PVW |
| Shift+Ctrl+F2	            | KAM 2 PVW |
| Shift+Ctrl+F3	            | KAM 3 PVW |
| Shift+Ctrl+F4	            | KAM 4 PVW |
| Shift+Ctrl+F5	            | KAM 5 PVW |
| Shift+Ctrl+Digit1	        | LIVE 1 til SS |
| Shift+Ctrl+Digit2	        | LIVE 2 til SS |
| Shift+Ctrl+Digit3	        | LIVE 3 til SS |
| Shift+Ctrl+Digit4	        | LIVE 4 til SS |
| Shift+Ctrl+Digit5	        | LIVE 5 til SS |
| Shift+Ctrl+Digit6	        | LIVE 6 til SS |
| Shift+Ctrl+Digit7	        | LIVE 7 til SS |
| Shift+Ctrl+Digit8	        | LIVE 8 til SS |
| Shift+Ctrl+Digit9	        | LIVE 9 til SS |
| Shift+Ctrl+Digit0	        | LIVE 10 til SS |
| Shift+Ctrl+KeyE	            | EVS 1 til SS |
| Shift+Ctrl+KeyA	            | ovl: ident OUT |
| Shift+Ctrl+KeyS	            | ovl: top OUT |
| Shift+Ctrl+KeyD	            | ovl: lower OUT |
| Shift+Ctrl+KeyF	            | ovl: headline OUT |
| Shift+Ctrl+KeyG	            | ovl: tema OUT |
| Ctrl+Alt+Shift+KeyF	        | EVS 1 til OVL inp 1 |
| Shift+Ctrl+F1	            | KAM 1 PVW |
| Shift+Ctrl+F2	            | KAM 2 PVW |
| Shift+Ctrl+F3	            | KAM 3 PVW |
| Shift+Ctrl+F4	            | KAM 4 PVW |
| Shift+Ctrl+F5	            | KAM 5 PVW |
| Ctrl+Alt+Shift+KeyG	        | serv til inp 2 |
| Shift+T	                    | serv til inp 1 |
| Shit+Alt+F1	                | KAM 1 til inp 3 |
| Shift+Alt+F2	            | KAM 2 til inp 3 |
| Shift+Alt+F3	            | KAM 3 til inp 3 |
| Shift+Alt+F4	            | KAM 4 til inp 3 |
| Shift+Alt+F5	            | KAM 5 til inp 3 |
| Shift+Alt+Digit1	        | LIVE 1 til inp 3 |
| Shift+Alt+Digit2	        | LIVE 2 til inp 3 |
| Shift+Alt+Digit3	        | LIVE 3 til inp 3 |
| Shift+Alt+Digit4	        | LIVE 4 til inp 3 |
| Shift+Alt+Digit5	        | LIVE 5 til inp 3 |
| Shift+Alt+Digit6	        | LIVE 6 til inp 3 |
| Shift+Alt+Digit7	        | LIVE 7 til inp 3 |
| Shift+Alt+Digit8	        | LIVE 8 til inp 3 |
| Shift+Alt+Digit9	        | LIVE 9 til inp 3 |
| Shift+Alt+Digit0	        | LIVE 10 til inp 3 |
| F1	                        | KAM 1 |
| F2	                        | KAM 2 |
| F3	                        | KAM 3 |
| F4	                        | KAM 4 |
| Shift+Ctrl+Alt+KeyC	        | KAM 5 |
| NumpadDivide	            | Take With Transition |
| NumpadSubtract	            | Take With Transition |
| NumpadAdd	                | Take With Transition |
| Space	                    | GFX Continue |
| Alt+Shift+KeyE	            | EVS 1 100% box 3 |
| Alt+Shift+KeyD	            | EVS 1 VO box 3 |
| Alt+Shift+KeyG	            | EVS 2 100% box 3 |
| KeyA	                    | Lower third segment adlibs |
| KeyS	                    | Lower third segment adlibs |
| KeyD	                    | Lower third segment adlibs |
| KeyF	                    | Lower third segment adlibs |
| KeyG	                    | Lower third segment adlibs |

## Other Information

This PR cannot be merged until @ianshade has finished his work on the keyboard map system, and requires the `blueprints-integration` from the `feat/SOF-510/actionTriggersWithKeyboardMap` branch of core, where his work is based.

This PR bumps the blueprints version to `1.7.0`.

### CODE REVIEW

**Comment by thomasslee97** [comment]
You'll probably wan't to start in `src/tv2_afvd_showstyle/migrations/index.ts` and `src/tv2_offtube_showstyle/migrations/index.ts` when doing your reviews.

**Review by skon-tv2 — APPROVED** [review]

File: src/tv2-common/hotkeys/hotkey-defaults.ts [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+	}),
+	global: literal<GlobalHotkeyAssignments>({
+		recallLast: {
+			DVE: 'F10',
+			Live: 'Ctrl+Alt+Shift+KeyB'
+		},
+		takeWithTransition: ['NumpadDivide', 'NumpadSubtract', 'NumpadAdd'],
+		camera: {
+			directCut: ['F1', 'F2', 'F3', 'F4', 'F5'],
+			queue: ['Ctrl+Shift+F1', 'Ctrl+Shift+F2', 'Ctrl+Shift+F3', 'Ctrl+Shift+F4', 'Ctrl+Shift+F5'],
```
Comment (ianshade) [inline src/tv2-common/hotkeys/hotkey-defaults.ts:25]: It should be `Control` instead of `Ctrl` https://github.com/nrkno/sorensen/blob/main/README.md

**Review by ianshade — CHANGES_REQUESTED** [review]

**Review by KvelaGorrrrnio — APPROVED** [review]
Good work! 🚀

**Review by leerberg — APPROVED** [review]
GJ 🚀

**Pull request merged**